packages/components/package.json

```
@@ -35,14 +35,24 @@
     "lint:js:fix": "eslint . --fix"
   },
   "dependencies": {
+    "@codemirror/commands": "^6.7.1",
+    "@codemirror/lang-go": "^6.0.1",
+    "@codemirror/lang-javascript": "^6.2.2",
+    "@codemirror/lang-json": "^6.0.1",
+    "@codemirror/lang-sql": "^6.8.0",
+    "@codemirror/language": "^6.10.3",
+    "@codemirror/state": "^6.4.1",
+    "@codemirror/view": "^6.33.0",
     "@ember/render-modifiers": "^2.0.5",
     "@ember/string": "^3.1.1",
     "@ember/test-waiters": "^3.1.0",
     "@embroider/addon-shim": "^1.8.7",
     "@floating-ui/dom": "^1.6.3",
     "@hashicorp/design-system-tokens": "^2.2.1",
     "@hashicorp/flight-icons": "^3.7.0",
+    "@lezer/highlight": "^1.2.1",
     "clipboard-polyfill": "^4.1.0",
+    "codemirror": "^5.65.12",
```

the main `codemirror` dependency must be removed (this is covered by `@codemirror/state` and `@codemirror/view`, as detailed in the RFC)
```suggestion
    "codemirror": "^5.65.12",
```